docs/Account/methods.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Methods
+---
+
+The SDK provides the following methods for a smart account.
+
+## BiconomySmartAccountV2
+### Create
+
+The `create()` method is used to create an instance of the Biconomy Smart Account V2. This method takes a configuration object and returns the smart account instance.
```

Comment:
under create you can mention that this is not a multi chain instance. to use smart account on another chain you need to instantiate another smart account API instance with configuration of that chain.